pkg/network/ovn_kubernetes.go

```diff
@@ -561,12 +579,12 @@ func shouldRenderIPsec(conf *operv1.OVNKubernetesConfig, bootstrapResult *bootst
 	// MachineConfig IPsec extensions are needed for the ipsec deployment except
```

Comment:
I believe the comment here is misleading and should mention EW ipsec, this is what the code does.